packages/client/components/TeamPrompt/WorkDrawer/GitHubIntegrationPanel.tsx

```diff
@@ -0,0 +1,101 @@
+import React, {useState} from 'react'
+import graphql from 'babel-plugin-relay/macro'
+import {useFragment} from 'react-relay'
+import {GitHubIntegrationPanel_meeting$key} from '../../../__generated__/GitHubIntegrationPanel_meeting.graphql'
+import clsx from 'clsx'
+import gitHubSVG from '../../../styles/theme/images/graphics/github-circle.svg'
+import GitHubClientManager from '../../../utils/GitHubClientManager'
+import useAtmosphere from '../../../hooks/useAtmosphere'
+import useMutationProps from '../../../hooks/useMutationProps'
+import GitHubIntegrationResultsRoot from './GitHubIntegrationResultsRoot'
+
+const GITHUB_QUERY_TABS: {key: 'issue' | 'pullRequest'; label: string}[] = [
+  {
+    key: 'issue',
+    label: 'Issues'
+  },
+  {
+    key: 'pullRequest',
+    label: 'Pull Requests'
+  }
+]
+
+interface Props {
+  meetingRef: GitHubIntegrationPanel_meeting$key
+}
+
+const GitHubIntegrationPanel = (props: Props) => {
+  const {meetingRef} = props
+  const meeting = useFragment(
+    graphql`
+      fragment GitHubIntegrationPanel_meeting on TeamPromptMeeting {
+        viewerMeetingMember {
+          teamMember {
+            teamId
+            integrations {
+              github {
+                isActive
+              }
+            }
+          }
+        }
+      }
+    `,
+    meetingRef
+  )
+
+  const teamMember = meeting.viewerMeetingMember?.teamMember
+
+  const [githubType, setGithubType] = useState<'issue' | 'pullRequest'>('issue')
+
+  const atmosphere = useAtmosphere()
+  const mutationProps = useMutationProps()
+
+  const authGitHub = () => {
+    teamMember && GitHubClientManager.openOAuth(atmosphere, teamMember.teamId, mutationProps)
```

Comment:
+1 this should never happen, but if it does it'd be good to _not_ silently do nothing